### PR TITLE
allow StringCrossFloatTransform to cross individual features

### PR DESCRIPTION
**Pull request by jq**
@deerzq @saurfang

### CODE REVIEW

File: core/src/main/java/com/airbnb/aerosolve/core/transforms/StringCrossFloatTransform.java [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
-import java.util.HashMap;
-import java.util.Set;
-import java.util.Map;
 
+import java.util.*;
+
+/*
+  if no prefix/keys provided, then cross whole family.
+  prefix used in string, and keys used in float.
+  to cross RAW features, use Features.RAW
```
Comment (deerzq) [inline core/src/main/java/com/airbnb/aerosolve/core/transforms/StringCrossFloatTransform.java]: please add more details about what is key, what is prefix, without actually looking at the code, it's not clear that prefix is for string family and key is for float feature. and it's also not clear what exactly is prefix, people may think it means the prefix of a feature name, but it's actually feature name.

**Comment by jq** [comment]
@deerzq thanks for review, updated!

File: core/src/main/java/com/airbnb/aerosolve/core/features/Features.java [hunk 2]
```diff
@@ -120,6 +120,19 @@ static void addStringFeature(
     }
   }
 
+  // string feature is concatenated by : the prefix before : is feature name
+  // RAW feature has no : so just return the RAW
+  // this is used in StringCrossFloatTransform so that
+  // we can cross Raw feature as well as other string features
+  public static String getStringFeatureName(String feature) {
+    String[] tokens =  feature.split(":");
```
Comment (deerzq) [inline core/src/main/java/com/airbnb/aerosolve/core/features/Features.java]: make ":" a constant?

**Comment by deerzq** [comment]
one more comment, LGTM

**Pull request merged**